app/views/contexts/summary/group.py

```diff
@@ -10,24 +16,24 @@ class Group:
     def __init__(
         self,
         *,
-        group_schema,
-        routing_path,
-        answer_store,
-        list_store,
-        metadata,
-        response_metadata,
-        schema,
-        location,
-        language,
-        progress_store,
-        return_to,
+        group_schema: Mapping[str, list[Any]],
```

Comment:
Group schema values aren't all lists. `Mapping` or `Mapping[str, Any]` would be appropriate. For schema dicts, we can't be that explicit due to dynamic nature of it. We will look at creating models for them in future.

Reply:
👍